### DISCUSSION TITLE
KServe keeps creating public ingress

### DISCUSSION THREAD

**Discussion by abdulvirta** [kubeflow/kfserving#2211] [Q&A]
Hi all, I'm using KServe to serve a model. I'm using RawDeployment. I already have ISTIO setup for exposing this service. The problem is that KServe keeps creating an Ingress with public IP. Is there anyway to just tell KServe not create an ingress or not do anything on the networking side of things as we are already managing separately.

**Comment by yuzisun** [comment]
I think we need to create an option for this to not provision the ingress.

**Reply by abdulvirta** [reply]
yes agree this would be great 👍 . Thanks for the follow up!
